src/Document.php

```diff
@@ -358,6 +359,12 @@ public function post_addItem()
                 'items_id'     => $this->input["items_id"]
             ]);
 
+            if ($this->input["itemtype"] == \Ticket::getTypeName()) {
+                $ticket = new Ticket();
+                $ticket->getFromDB($this->input["items_id"]);
+                $this->updateParentStatus($ticket, $this->fields);
+            }
+
```

Comment:
I think the same applies to changes and problems.
```suggestion
            $itilobject = new $this->input["itemtype"]();
            $itilobject->getFromDB($this->input["items_id"]);
            $this->updateParentStatus(itilobject, $this->input);
```

Reply:
I've made the suggested changes by adding the missing `$` on `itilobject` l.367. Thank !